pandas/tests/window/test_numba.py

```diff
@@ -56,9 +56,7 @@ class TestEngine:
     @pytest.mark.parametrize("jit", [True, False])
     def test_numba_vs_cython_apply(self, jit, nogil, parallel, nopython, center, step):
         def f(x, *args):
-            arg_sum = 0
-            for arg in args:
-                arg_sum += arg
+            arg_sum = np.sum(args)
```

Comment:
This is a test and the function is pushed to cython. So I don't think there is any need to change this

Reply:
Thanks for opinion. NumPy is utilized in many places of this script. This could be a place whether it has missed to utilize it.

Reply:
This might have been on purpose. If there is no need to change tests, we try to avoid this.

cc @mroeschke

Reply:
Tests are failing

Reply:
Yeah this is a contrived example for testing numba compilation with `*args`. This should be left as is